docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -160,5 +160,9 @@ <h2>The result</h2>
 			&lt;/body&gt;
 		&lt;/html&gt;
 		</code>
+		
+		<h2>An alternative way to install three.js using NPM:</h2>
+		
+		<code>$ npm install three --save-dev</code>
```

Comment:
```html
<code>$ npm install three</code>
```

Reply:
actually I think it should be <code>$ npm install three --save</code>, or <code>$ npm i three -d</code>.--save or -d can put three into dependencies. We don't need to use -dev, because it will add three into dev-dependencies

Reply:
And maybe you can also add <code>$ yarn add three</code>,